### PR TITLE
Add namespace consideration

### PR DESCRIPTION
**Pull request by yhyakuna**
🔍 [Deploy preview](https://vault-8ytre2o97-hashicorp.vercel.app/vault/docs/ui/custom-messages)

This is a draft to bounce off ideas to incorporate [Finn's feedback](https://github.com/hashicorp/vault/pull/25435#discussion_r1499973087).

**Background:**

While this feature is available for Community Edition as well, the concept of namespaces only apply to Vault Enterprise and HCP Vault. So, I decided to add **Namespaces consideration** section immediately after going through the basic steps. --> Community users can just skip this portion.

### CODE REVIEW

**Comment by yhyakuna** [comment]
@kiannaquach @finnstech  I created a PR to address **namespaces**.  Feel free to suggest if this wasn't what you were thinking of.

**Review by kiannaquach — APPROVED** [review]
LGTM! Thanks for helping with this!

**Comment by finnstech** [comment]
There are some typos, can we reopen this PR so I can add a review? Thanks!

File: website/content/docs/ui/custom-messages.mdx [hunk 1]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
 
 1. Click **Create message** to save your new message.
 
+### Namespaces consideration
+
+When you are running Vault Enterprise or [HCP Vault](/hcp/docs/vault) clusters,
+the custom messages created on a [namespace](/vault/docs/enterprise/namespaces)
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:70]: This should say "the custom messages create on [namespace]" (remove the "a")

File: website/content/docs/ui/custom-messages.mdx [hunk 2]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
...
+are inherited by its child namespaces. 
+
+For example, your cluster has the following namespace hierarchy.
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:73]: "For example, a cluster with the following namespace hierarchy:"

File: website/content/docs/ui/custom-messages.mdx [hunk 3]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
...
+    ├── finance
+    └── marketing
+        ├── digital-marketing
+        └── events
+```
+
+</CodeBlockConfig>
+
+Custom messages created on the `admin` namespace apply to all child namespaces:
+`fiance`, `marketing`, `marketing/digital-marketing` and `marketing/events`.
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:88]: "fiance" should be "finance"

File: website/content/docs/ui/custom-messages.mdx [hunk 4]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
...
+
+To create a custom message intended to the marketing team but not for `finance`,
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:90]: "intended for" not "intended to"

File: website/content/docs/ui/custom-messages.mdx [hunk 5]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
...
+you must log into the `marketing` namespace to create a message.
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:91]: remove "you must"

File: website/content/docs/ui/custom-messages.mdx [hunk 6]
```diff
@@ -64,6 +64,51 @@ Use custom banners and modals in the Vault UI to share system-wide alerts for al
...
+1. Select the **+ Create message** button in the toolbar to open  and fill out
+   the **Create message** form.
+
+   ![Custom message](/img/ui-custom-msg.png)
+
+1. Click **Preview** to see how your message will appear to users.
+
+1. Click **Create message** to save your new message.
+
+The message banner will only appear when a user logged into the `marketing`
```
Comment (finnstech) [inline website/content/docs/ui/custom-messages.mdx:107]: "when a user logs in to" (not "logged into")

**Review by finnstech — COMMENTED** [review]
Made some edit suggestions to address typos

**Pull request merged**